### PR TITLE
Bdtt 52 varun updated scraper ms error handling

### PR DESCRIPTION
**Pull request by VaruneshBala**
# Description

Fixes [Jira ticket 52]()

List all acceptance criterias of user story and indicate which has been solved previously and which is solved currently
- [x] XXX (solved)
- [] XXX (solving)
- [ ]

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactor
- [ ] Migration

# Checklist:

- [x] I have added the appropriate labels to my PR
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: services/backend/scrapers/Cordelia.py [hunk 1]
```diff
@@ -89,6 +97,12 @@ def track(tracking_type, tracking_identifier):
     finally:
         driver.close()
         
-    
+
+def restart_microservice():
```
Comment (pandaponn) [inline services/backend/scrapers/Cordelia.py:101]: Do we need to re-create the container or can we just re-call the scraper endpoint? Re-creating containers seem a bit extreme and this wouldn't work for our hosted solution, right?

File: docker-compose.yml [hunk 2]
```diff
@@ -203,6 +203,36 @@ services:
     shm_size: '1gb'
     ports:
       - 8083:8083
+  
+  # scraper_cosc:
```
Comment (pandaponn) [inline docker-compose.yml:207]: Commented out because they are not required?

**Comment by VaruneshBala** [comment]
@pandaponn pls approve thank you. made all the changes as per our last call

**Review by pandaponn — APPROVED** [review]
LGTM 👍🏻

**Pull request merged**